Source: Arne Janbu <arnej@ampheus.de>
Upstream: Not yet sent to upstream
Reason: Put libs in /usr/lib*, not in /lib*

--- qt-creator-1.1.0-src/src/qworkbench.pri_old	2009-06-10 01:10:30.503892213 +0200
+++ qt-creator-1.1.0-src/src/qworkbench.pri	2009-06-10 01:09:30.711882268 +0200
@@ -7,9 +7,9 @@
 }
 
 linux-*-64 {
-    IDE_LIBRARY_BASENAME = lib64
+    IDE_LIBRARY_BASENAME = /usr/lib64
 } else {
-    IDE_LIBRARY_BASENAME = lib
+    IDE_LIBRARY_BASENAME = /usr/lib
 }
 
 equals(TEST, 1) {
